submissions/titilayoOlutola/HTML/Html-task.html

```diff
@@ -21,37 +21,15 @@ <h4> <em>your-library-on-the-go</em></h4>
         <ul>
             <li><a href="#">Home</a></li>
             <li><a href="#">Search</a></li>
-            <li><a href="#">Contact Us</a></li>
+            <li><a href="C:\Users\laptop\Desktop\Dufuna-Fem\submissions\titilayoOlutola\html-forms\contact-form.html">Contact Us</a></li>
```

Comment:
This `href="C:\Users\laptop\Desktop\Dufuna-Fem\submissions\titilayoOlutola\html-forms\contact-form.html"` is wrong and not a relative path. If I clone your work it means I can't visit this page because on my computer I don't have `"C:\Users\laptop\Desktop\..."`. I will like you to figure out how to do this better

Reply:
I copied the path for my contact form and inserted it as the href factor. How do I go about it in the right way?